pyproject.toml

```
@@ -49,7 +48,7 @@ packages = [{ include = "flexmock", from = "src" }]
 "Issue Tracker" = "https://github.com/flexmock/flexmock/issues"
 
 [tool.poetry.dependencies]
-python = "^3.7.1"
+python = "^3.8.1"
```

`3.8.0` should probably be okay. Earlier we had to bump it up to `3.7.1` because some dependency needed that version.
```suggestion
python = "^3.8.0"
```